onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -1069,6 +1069,10 @@ ::std::vector<::std::basic_string<ORTCHAR_T>> GetParameterStrings() {
 #if !defined(_WIN32) && !defined(USE_OPENVINO)
     paths.push_back("/data/onnx");
 #endif
+
+    if (Env::Default().FolderExists(ORT_TSTR("simple"))) {
+      paths.push_back(ORT_TSTR("simple"));
```

Comment:
What creates the 'simple' folder?

Reply:
The name is hard coded in ONNX's test generating script.